utils/build/build.sh

```diff
@@ -289,14 +289,7 @@ build() {
                 fi
             fi
         elif [[ $IMAGE_NAME == lambda-proxy ]]; then
-            docker buildx build \
-                --build-arg BUILDKIT_INLINE_CACHE=1 \
-                --load \
-                --progress=plain \
-                -f utils/build/docker/lambda-proxy.Dockerfile \
-                -t datadog/system-tests:lambda-proxy \
-                $EXTRA_DOCKER_ARGS \
-                .
+            echo "Building lambda-proxy is now handled by utils/build/build_lambda_proxy.sh"
```

Comment:
`build.sh` is the entry point for building images in system-tests, and is used in many context, we need to keep this feature here.

Reply:
Good catch! I was using it to grep the logs and spot other jobs that would use it. Would restoring the behavior by calling the added `./utils/build/build_lambda_proxy.sh` script works for you?